substrate/frame/staking-async/src/pallet/impls.rs

```diff
@@ -880,6 +885,98 @@ impl<T: Config> Pallet<T> {
 	pub fn api_pending_rewards(era: EraIndex, account: T::AccountId) -> bool {
 		Eras::<T>::pending_rewards(era, &account)
 	}
+
+	/// Calculates the total stake of the lowest portion validators and stores it for the planned
+	/// era.
+	pub(crate) fn calculate_lowest_total_stake(era: EraIndex) {
+		// Only calculate if unbonding queue params have been set.
+		if let Some(params) = UnbondingQueueParams::<T>::get() {
+			// Determine the total stake from the lowest portion of validators and persist for the
+			// era.
+			let mut total_stake: Vec<_> =
```

Comment:
```suggestion
			let mut validator_total_stakes: Vec<_> =
```

This can be mistaken with the `.sum` of all of them, which we already track in `ErasTotalStake`.

Reply:
Done in [2079d89](https://github.com/paritytech/polkadot-sdk/pull/8298/commits/2079d899504b69781d0374a7836dd2c24c68dacd).